FLEDGE.md

```diff
@@ -151,13 +150,20 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
                         '*': 150,
                         ...},
+  'componentAuctions': [
+    {'seller': 'https://www.some-other-ssp.com',
+      ...},
+    ...
+  ]
 };
 const auctionResultPromise = navigator.runAdAuction(myAuctionConfig);
 ```
 
 
 This will cause the browser to execute the appropriate bidding and auction logic inside a collection of dedicated worklets associated with the buyer and seller domains.  The `auctionSignals`, `sellerSignals`, and `perBuyerSignals` values will be passed as arguments to the appropriate functions that run inside those worklets — the `auctionSignals` are made available to everyone, while the other signals are given only to one party.
 
+In some cases, multiple SSPs may want to run their own auction and send the winner up to another auction. To facilitate these "component auctions", `componentAuctions` can optionally contain additional auction configurations for each seller's "component auction".  The winning bid of each of these "component auctions" will be passed to the top-most auction.  How bids are scored in this case is further described in [2.4 Scoring Bids in Component Auctions](#24-scoring-bids-in-component-auctions).
+
 The returned `auctionResultPromise` object is _opaque_: it is not possible for any code on the publisher page to inspect the winning ad or otherwise learn about its contents, but it can be passed to a Fenced Frame for rendering.  (The [Fenced Frame Opaque Source explainer](https://github.com/shivanigithub/fenced-frame/blob/master/OpaqueSrc.md) has initial thoughts about how this could be implemented.)  If the auction produces no winning ad, the return value can also be null, although this non-opaque return value leaks one bit of information to the surrounding page.  In this case, for example, the seller might choose to render a contextually-targeted ad.
 
 Optionally, `perBuyerTimeouts` can be specified to restrict the runtime (in milliseconds) of particular buyer's bidding scripts. If no value is specified for a particular buyer, a default timeout of 50 ms will be selected. Any `perBuyerTimeouts` higher than 500 ms will be clamped to 500 ms. A key of `'*'` is used to change the default of unspecified buyers.
```

Comment:
How does `perBuyerTimeouts` work with components?  Is there a way to apply a timeout to a component auction?

Reply:
They are only scoped to that AuctionConfig.  I've added a general statement about the scoping of fields here.  Component auctions have their own perBuyerTimeouts field, which applies only to them.

Reply:
Could there also be a `perComponentSellerTimeouts` field in the top level config?  If someone is orchestrating a multilevel auction, they probably need the ability to control how long components can run for.

Reply:
That is, a timeout that applies to the entire component auction run by that component seller.

Reply:
How about adding a sellerTimeout or somesuch to each AuctionConfig (which also lets the top-level seller worklet timeout if it's taking too long)?  I've currently deliberately avoided having fields of the top-level auction apply to the component auctions, though I'm certainly open to changing that.

Reply:
That works!

Reply:
I think this belongs in another PR, so I'm leaving this comment open until I get to it.

Reply:
Marking this is resolved, since Qingxin has a PR out for this now.